docs/tutorials/libraries/symbols/odr-global.rst

```diff
@@ -130,7 +130,7 @@ put the code under ``if`` condition:
 .. code-block:: cmake
 
   if(NOT EXISTS "${CMAKE_TOOLCHAIN_FILE}")
-    set(CMAKE_CXX_STANDARD 11)
-    set_target_properties(boo PROPERTIES CXX_STANDARD 14)
+    set(CMAKE_CXX_STANDARD 11) # set a global minimum standard
+    set_target_properties(boo PROPERTIES CXX_STANDARD 14) # set a standard for a target
```

Comment:
You just explained what command do, not why.

I mean something like this:
```
If you still want to set global flags locally for any reason then at least
put the code under ``if`` condition. For example let's set C++11 for
all targets in project and C++14 for target ``boo``:
```

Reply:
better? :)

Reply:
Yep, thanks 👍